### PR TITLE
Resolve compilation failures due to version errors.

### PR DESCRIPTION
**Pull request by 99Kies**
- [x] Resolve compilation failures due to version errors. 
and update on dao-core : <https://github.com/DoraFactory/dorafactory-dao-core/pull/1>
- [x] Complete README info.

### CODE REVIEW

**Comment by tinydjp** [comment]
Hi @99Kies thanks for your commit, may I know what OS version did you use? I compiled successfully on my Mac M1 laptop, maybe there'll be some compatibility issues

**Comment by 99Kies** [comment]
> Hi @99Kies thanks for your commit, may I know what OS version did you use? I compiled successfully on my Mac M1 laptop, maybe there'll be some compatibility issues

wsl2-ubuntu20.04

The problem here should not be the difference between THE M1 system and other systems (they just have different system commands).  

-----

The problem is that the dorafactory-dao-core version is `3.0.0`, but you imported `0.1.0`
image
image

**Comment by tinydjp** [comment]
okok, cuz the commits are too many, just checked again should be like you said, I'll assign the reviewer. @VegeBun-csj

**Comment by VegeBun-csj** [comment]
> > Hi @99Kies thanks for your commit, may I know what OS version did you use? I compiled successfully on my Mac M1 laptop, maybe there'll be some compatibility issues
> 
> wsl2-ubuntu20.04
> 
> The problem here should not be the difference between THE M1 system and other systems (they just have different system commands).
> 
> The problem is that the dorafactory-dao-core version is `3.0.0`, but you imported `0.1.0` image image

thank you for this commit. we update the version of dorafactory-node and dorafactory-dao-core(https://github.com/DoraFactory/dorafactory-dao-core/tree/fix-dependency) to 4.0.0-dev. you can run the chain by the fix. if success, push your latest branch again.i will merge.thanks

File: pallets/moloch-v2/Cargo.toml [hunk 1]
```diff
@@ -1,93 +1,93 @@
...
-    'frame-support/std',
-    'frame-system/std',
-	'pallet-balances/std',
-    'pallet-timestamp/std',
-    'sp-timestamp/std',
-	'sp-runtime/std',
+    "codec/std",
+    "frame-support/std",
+    "frame-system/std",
+	"pallet-balances/std",
```
Comment (VegeBun-csj) [inline pallets/moloch-v2/Cargo.toml:40]: adjust the format

File: pallets/quadratic-funding/Cargo.toml [hunk 2]
```diff
@@ -1,99 +1,99 @@
...
+runtime-benchmarks = ["frame-benchmarking"]
 std = [
-    'codec/std',
-    'frame-support/std',
-    'frame-system/std',
-        'pallet-dao-core/std',
+    "codec/std",
+    "frame-support/std",
+    "frame-system/std",
+        "pallet-dao-core/std",
```
Comment (VegeBun-csj) [inline pallets/quadratic-funding/Cargo.toml:41]: adjust the format

**Comment by 99Kies** [comment]
> > > Hi @99Kies thanks for your commit, may I know what OS version did you use? I compiled successfully on my Mac M1 laptop, maybe there'll be some compatibility issues
> > 
> > 
> > wsl2-ubuntu20.04
> > The problem here should not be the difference between THE M1 system and other systems (they just have different system commands).
> > The problem is that the dorafactory-dao-core version is `3.0.0`, but you imported `0.1.0` image image
> 
> thank you for this commit. we update the version of dorafactory-node and dorafactory-dao-core(https://github.com/DoraFactory/dorafactory-dao-core/tree/fix-dependency) to 4.0.0-dev. you can run the chain by the fix. if success, push your latest branch again.i will merge.thanks

@VegeBun-csj ok, I'll contribute in my free time.

**Comment by 99Kies** [comment]
> > > Hi @99Kies thanks for your commit, may I know what OS version did you use? I compiled successfully on my Mac M1 laptop, maybe there'll be some compatibility issues
> > 
> > 
> > wsl2-ubuntu20.04
> > The problem here should not be the difference between THE M1 system and other systems (they just have different system commands).
> > The problem is that the dorafactory-dao-core version is `3.0.0`, but you imported `0.1.0` image image
> 
> thank you for this commit. we update the version of dorafactory-node and dorafactory-dao-core(https://github.com/DoraFactory/dorafactory-dao-core/tree/fix-dependency) to 4.0.0-dev. you can run the chain by the fix. if success, push your latest branch again.i will merge.thanks

About dorafactory-dao-core of the problem, before I submit a pr https://github.com/DoraFactory/dorafactory-dao-core/pull/1

**Comment by 99Kies** [comment]
@VegeBun-csj  fixed

**Comment by 99Kies** [comment]
https://github.com/DoraFactory/dorafactory-node/pull/2
https://github.com/DoraFactory/dorafactory-dao-core/pull/1
@VegeBun-csj

**Comment by 99Kies** [comment]
This needs to be upgraded along with `dao-core`. @VegeBun-csj

**Pull request merged**